netlogo-gui/src/main/lab/gui/Supervisor.scala

```diff
@@ -66,6 +66,12 @@ class Supervisor(
   private val workerThread = new Thread(runnable, "BehaviorSpace Worker")
   private val progressDialog = new ProgressDialog(dialog, this, saveProtocol)
   private val exporters = new ListBuffer[Exporter]
+  // private val exporterFileNames = new HashMap[Exporter, String]
```

Comment:
Don't keep commented out code. If there is a question, or something to tell future code maintainers put it in a comment.